cmd/crank/main.go

```diff
@@ -55,13 +55,21 @@ func (v verboseFlag) BeforeApply(ctx *kong.Context) error { //nolint:unparam //
 	return nil
 }
 
+// The top-level crossplane CLI.
 var cli struct {
-	Version versionFlag `short:"v" name:"version" help:"Print version and quit."`
-	Verbose verboseFlag `name:"verbose" help:"Print verbose logging statements."`
+	// Subcommands and flags will appear in the CLI help output in the same
+	// order they're specified here. Keep them in alphabetical order.
+
+	// Subcommands.
+	XPKG xpkg.Cmd `cmd:"" help:"Manage Crossplane packages."`
 
-	XPKG xpkg.Cmd `cmd:"" help:"Crossplane package management."`
+	// The alpha and beta subcommands are intentionally in a separate block. We
+	// want them to appear after all other subcommands.
+	Beta beta.Cmd `cmd:"" help:"Beta commands."`
 
-	Beta beta.Cmd `cmd:"" help:"Beta features. WARN: May be changed or deprecated in a future release"`
+	// Flags.
+	Verbose verboseFlag `name:"verbose" help:"Print verbose logging statements."`
+	Version versionFlag `short:"v" name:"version" help:"Print version and quit."`
 }
```

Comment:
This renders as:

```
Usage: crossplane <command>

A command line tool for interacting with Crossplane.

Commands:
  xpkg build        Build a new package.
  xpkg install      Install a package from a package registry.
  xpkg login        Login to the default package registry.
  xpkg logout       Logout of the default package registry.
  xpkg push         Push a package to a package registry.
  xpkg update       Update the version of an installed package.
  beta render       Render a composite resource (XR).
  beta xpkg init    Initialize a new package from a template.

Flags:
  -h, --help       Show context-sensitive help.
      --verbose    Print verbose logging statements.
  -v, --version    Print version and quit.

Run "crossplane <command> --help" for more information on a command.
```

I'm intentionally not saying only "package" at this level. I believe:

* It's the `xpkg` subcommand of the Crossplane CLI. Saying "Crossplane package" feels quite redundant.
* Similar for "xpkg package". Feels a bit like writing ATM machine.